base/testfiles-lthooks/lthooks-018.lvt

```diff
@@ -15,8 +15,8 @@
 
 \START
 
-\AddToHook{env/before/quote}[my1]{\dotfill before 1\dotfill\par}
+\AddToHook{env/quote/before}[my1]{\dotfill before 1\dotfill\par}
 
-\LogHook{env/before/quote}
+\LogHook{env/quote/before}
```

Comment:
I wonder what this was supposed to test in the past. "env" always had its variable part in position 2  ... so maybe the test was supposed to test for an input error and should not be altered. After all, 019 has env/quote/before in the right order

Reply:
I tracked this test file down to commit 61460202 in the AccessiblePDF repository, which was related to `\typeout{\par}` (triggered by a `\ShowHook`), so the intent of the test didn't change.  It was just a typo back then.

Reply:
then maybe we should add a note to the test what it is testing :-) as it looks a little useless on first glance

Reply:
I added a better explanation